app/test_split_integer.py

```diff
@@ -2,20 +2,25 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert (sum(split_integer(8, 1)) == 8), \
+        "Sum of parts for split_integer(8, 1) should be 8"
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert (split_integer(6, 2) == [3, 3]),\
```

Comment:
Pls try to remove this breaking lines everywhere ( \ ). Use more beautiful variants : )